regulation-worker/internal/delete/batch/batch.go

```diff
@@ -308,7 +309,10 @@ type BatchManager struct {
 }
 
 func (*BatchManager) GetSupportedDestinations() []string {
-	return supportedDestinations
+	if config.Default.GetBool("REGULATION_WORKER_BATCH_DESTINATIONS_ENABLED", false) {
```

Comment:
I think the default value for it should be `true`. Disabling it should be a special case.

Reply:
IMO we cannot enable it until we have proper guards in the worker to make sure that jobs will not take forever to complete.